### ISSUE TITLE
Conversion of input/output parameters

### ISSUE THREAD

**Issue by MerlinSmiles** [microsoft/Qcodes#57]
Related to isue #56 there is another problem, internal conversion of values.

Sometimes the data you get or set in a measurement is not what that data actually represents.
For instance:
- One might have a voltage divider at the output of a voltage source. Here I would like a conversion factor somewhere in the instrument that takes care of that. At the end of the day I'd like to say: `bias.set(1e-3)` but the instrument might have to output `1.213V`.
- There could be an analog instrument in from of the instrument you measure. For instance a Current amplifier that translates Amperes to Volts, When I measure the voltage this represents the current. At the end of the day I'd like to say `current.get()` and get something in Amperes.

Both cases deal with manual additions/changes to instruments, or rather the representation of their values.
Should we have, Is there, or is there planned to have a standardized conversion of this?

**Comment by giulioungaretti** [comment]
@MerlinSmiles  can you rephrase this?

**Comment by MerlinSmiles** [comment]
@giulioungaretti Not sure how to rephrase, what does not make sense?
Its kindof related to the signal chan thing from @WilliamHPNielsen

**Comment by giulioungaretti** [comment]
I am confused by this "Sometimes the data you get or set in a measurement is not what that data actually represents. "

**Comment by MerlinSmiles** [comment]
Lets say you have a DMM measuring the output of and current amplifier, you measure voltage but it represents current.
Right now I use i.e. the parameter that is inside the ithaco driver or the parameter that is inside the SR560 diver, but the whole conversion thing is much more general and IMHO we should not write the same thing for each situation.
Right now I use the ithaco thing for a totally different instrument.

**Comment by jenshnielsen** [comment]
I think this is mostly solved by the parameter aliases in the station configurator i.e. bx in this example https://github.com/qdev-dk/qdev-wrappers/blob/master/examples/station_configurator/exampleConfig.yaml 
We plan to move the configurator into qcodes main

**Comment by astafan8** [comment]
... or by [DelegateParameter class](https://github.com/qdev-dk/qdev-wrappers/blob/3bf23dcf5b5fd184e207ff80c1e1050c5c49ebc8/qdev_wrappers/parameters.py#L4) which will also moved to qcodes main soon (with some rework).

**Comment by WilliamHPNielsen** [comment]
The future looks bright 😎
